.travis.yml

```diff
@@ -4,32 +4,25 @@ python:
   - 3.8
   - 3.7
   - 3.6
-  - 3.5
-  - 2.7
 cache: pip
 env:
   global:
     - PYTHONWARNINGS=default,ignore::PendingDeprecationWarning,ignore::ResourceWarning
+    - DJANGO_DATABASE_USER_POSTGRES=postgres
+    - DJANGO_DATABASE_USER_MYSQL=travis
```

Comment:
I don't think these environmental variables are used by django-watson. I take it they're taken from the `travis.yml` in `django-reversion`?

So I think these can be taken out.

Reply:
You are right, I copied them from `django-reversion` for the sake of full travis config match, but they are not affecting the build. They are removed now.